### PR TITLE
+go-arch-lint

### PR DESCRIPTION
**Pull request by alexandregv**
- [x] do not edit `README.md` directly
- [x] `page.yaml` updated
- [x] `go generate` run

Thank you for contribution ! 🤝

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@
    + [ Explore dependencies with `goda`](#-explore-dependencies-with-goda)
    + [ Explore dependencies interactively with `spaghetti`](#-explore-dependencies-interactively-with-spaghetti)
    + [ Use `go mod` directives](#-use-go-mod-directives)
+   + [ Enforce Go code architecture with `go-arch-lint` with `go-arch-lint`](#-enforce-go-code-architecture-with-go-arch-lint-with-go-arch-lint)
```
Comment (nikolaydubina) [inline README.md:64]: yeah, you see name repeats twice.

File: page.yaml [hunk 2]
```diff
@@ -593,6 +593,16 @@ groups:
           exclude example.com/old/thing v1.2.3
           replace example.com/bad/thing v1.4.5 => example.com/good/thing v1.4.5
           retract [v1.9.0, v1.9.5]
+      - title: Enforce Go code architecture with `go-arch-lint`
```
Comment (nikolaydubina) [inline page.yaml:596]: you don't need to put name like this. `with <name>` is added automatically
```suggestion
      - title: Enforce Go code architecture
```

File: page.yaml [hunk 3]
```diff
@@ -687,6 +697,16 @@ groups:
         name: goexplorer
         author: https://github.com/ofabry
         example_image_url: https://github.com/ofabry/goexplorer/raw/master/images/screen.png
+      - title: Make D2 graph of architecture and dependencies with `go-arch-lint graph`
```
Comment (nikolaydubina) [inline page.yaml:700]:
```suggestion
      - title: Make D2 graph of architecture and dependencies
```

**Review by nikolaydubina — CHANGES_REQUESTED** [review]

**Comment by nikolaydubina** [comment]
could you remove name from title and re run go generate? good find btw!

File: README.md (hunk 1 above)
Comment (alexandregv) [reply README.md:64]: Oh I didn't see that! Thanks

**Review by nikolaydubina — APPROVED** [review]

**Pull request merged**